### PR TITLE
Constants for all keywords

### PR DESCRIPTION
**Pull request by PetrPytelka**
All keywords have its constants. It is easier to find usage of the keyword/constant in the code.

### CODE REVIEW

File: src/main/java/com/scriptbasic/interfaces/ScriptBasicKeyWords.java [hunk 1]
```diff
@@ -5,11 +5,61 @@
  * date June 26, 2012
  */
 public interface ScriptBasicKeyWords {
-    String[] BASIC_KEYWORDS = new String[]{"for", "end", "next", "let", "if",
-            "endif", "then", "while", "wend", "else", "elseif", "repeat",
-            "until", "not", "false", "true", "and", "or", "div", "use", "from",
-            "as", "to", "step", "method", "is", "rem", "sub", "endsub",
-            "return", "print", "global", "local", "call", "sentence"};
+    public static final String KEYWORD_AND = "and";
```
Comment (verhas) [inline src/main/java/com/scriptbasic/interfaces/ScriptBasicKeyWords.java]: Because this is an `interface` the keywords `public`, `static` and `final` are meaningless. A field in an interface is by default `public static final` and it cannot be anything else.

Please delete these keywords.

File: src/main/java/com/scriptbasic/interfaces/ScriptBasicKeyWords.java [hunk 2]
```diff
@@ -5,11 +5,61 @@
...
+    public static final String KEYWORD_SUB = "sub";
+    public static final String KEYWORD_THEN = "then";
+    public static final String KEYWORD_TO = "to";
+    public static final String KEYWORD_TRUE = "true";
+    public static final String KEYWORD_UNTIL = "until";
+    public static final String KEYWORD_USE = "use";
+    public static final String KEYWORD_WEND = "wend";
+    public static final String KEYWORD_WHILE = "while";
+
+    // Collection of all keywords, used by BasicKeywordRecognizer
```
Comment (verhas) [inline src/main/java/com/scriptbasic/interfaces/ScriptBasicKeyWords.java]: Use JavaDoc instead of one-line comments

File: src/main/java/com/scriptbasic/interfaces/ScriptBasicKeyWords.java [hunk 3]
```diff
@@ -5,11 +5,61 @@
...
+    //
+    // Note: 
+    // true, false (KEYWORD_TRUE, KEYWORD_FALSE) are constants thus not
+    // included in the collection of keywords
+    String[] BASIC_KEYWORDS = new String[]{KEYWORD_FOR, KEYWORD_END, KEYWORD_NEXT,
```
Comment (verhas) [inline src/main/java/com/scriptbasic/interfaces/ScriptBasicKeyWords.java]: Could you be more consistent with the formatting?

File: src/main/java/com/scriptbasic/syntax/BasicSyntaxAnalyzer.java [hunk 4]
```diff
@@ -1,6 +1,12 @@
 package com.scriptbasic.syntax;
 
-import com.scriptbasic.interfaces.*;
+import com.scriptbasic.interfaces.AnalysisException;
```
Comment (verhas) [inline src/main/java/com/scriptbasic/syntax/BasicSyntaxAnalyzer.java]: I do not like wild card imports, but ScriptBasic uses that. I believe that consistency in code is very important. If you want to change all import wildcards to import statements that individually import the classes I am happy with that. The only caveat is that it should be a separate pull request and should do it for all files.

Btw: it is fairly easy to change if you have IntelliJ. IntelliJ does that for you in a single menu click.

As for this pull request, as usual for all pull requests: keep it to contain only the changes that are strictly related to the actual request.

File: src/main/java/com/scriptbasic/syntax/commands/CommandAnalyzerMethod.java [hunk 5]
```diff
@@ -1,16 +1,17 @@
 package com.scriptbasic.syntax.commands;
 
+import java.util.ArrayList;
```
Comment (verhas) [inline src/main/java/com/scriptbasic/syntax/commands/CommandAnalyzerMethod.java]: Please try not to rearrange the imports. It is a kind of noise in the pull request. I was searching 30sec why `ArrayList` was introduced to the code when it is only replacing string literals to constants before I realized that it is only the rearrangement of the imports.

File: src/main/java/com/scriptbasic/syntax/expression/BasicExpressionAnalyzer.java [hunk 6]
```diff
@@ -1,12 +1,29 @@
 package com.scriptbasic.syntax.expression;
 
-import com.scriptbasic.context.Context;
-import com.scriptbasic.executors.operators.*;
-
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
+import com.scriptbasic.context.Context;
```
Comment (verhas) [inline src/main/java/com/scriptbasic/syntax/expression/BasicExpressionAnalyzer.java]: Please do it in a separate pull request.

File: src/main/java/com/scriptbasic/syntax/expression/BasicTagAnalyzer.java [hunk 7]
```diff
@@ -1,19 +1,32 @@
...
+
 import com.scriptbasic.context.Context;
 import com.scriptbasic.errors.BasicInterpreterInternalError;
 import com.scriptbasic.executors.operators.AbstractUnaryOperator;
 import com.scriptbasic.executors.operators.UnaryOperatorMinus;
 import com.scriptbasic.executors.operators.UnaryOperatorNot;
 import com.scriptbasic.executors.operators.UnaryOperatorPlus;
-import com.scriptbasic.executors.rightvalues.*;
-import com.scriptbasic.interfaces.*;
+import com.scriptbasic.executors.rightvalues.AbstractPrimitiveRightValue;
```
Comment (verhas) [inline src/main/java/com/scriptbasic/syntax/expression/BasicTagAnalyzer.java]: Please do it in a separate pull request.

File: src/main/java/com/scriptbasic/syntax/expression/BasicTagAnalyzer.java [hunk 8]
```diff
@@ -42,7 +55,7 @@
     static {
         unaryOperatorMap.put("+", UnaryOperatorPlus.class);
```
Comment (verhas) [inline src/main/java/com/scriptbasic/syntax/expression/BasicTagAnalyzer.java]: Perhaps this pull request could also assign a constant name to the unary and binary operators as well. Just a suggestion.

**Comment by PetrPytelka** [comment]
Thank you for review. I tried to resolve all issues.
